glfw/src/Window.zig

```diff
@@ -381,7 +437,10 @@ pub inline fn hint(h: Hint, value: anytype) Error!void {
 /// @thread_safety This function must only be called from the main thread.
 ///
 /// see also: window_creation, glfw.Window.destroy
-pub inline fn create(width: usize, height: usize, title: [*c]const u8, monitor: ?Monitor, share: ?Window) Error!Window {
+pub inline fn create(width: usize, height: usize, title: [*c]const u8, monitor: ?Monitor, share: ?Window, hints: Hints) Error!Window {
+    try hints.set();
+    try Window.defaultHints();
```

Comment:
I believe that this is wrong? It would set the user-specified hints, then overwrite them with the defaults?

Reply:
You are correct, that code is wrong. That probably explains why I was getting some strange results prototyping those hint-attribute tests I mentioned.